src/Microsoft.DotNet.XHarness.CLI/Commands/Apple/AppleAppCommand.cs

```diff
@@ -51,6 +52,7 @@ protected sealed override async Task<ExitCode> Invoke(Extensions.Logging.ILogger
 
             var diagnosticsData = serviceProvider.GetRequiredService<IDiagnosticsData>();
             diagnosticsData.Target = Arguments.Target.Value.AsString();
+            diagnosticsData.IsDevice = !Arguments.Target.Value.Platform.IsSimulator();
```

Comment:
May be it would be better to align naming with apple command - either `IsSimulator` or `IsDevice` for both

Reply:
Yeah, that would be nice, but unfortunately, I cannot change `IsSimulator` to `IsDevice` because Xamarin depends on this and I prefer to have `IsDevice` because it makes more sense for example in Apple where MacCatalyst has `IsSimulator == true` while it is not a simulator and it is not a device. So I think it would be better to have IsDevice even in this repo. But at least I make it less confusing down the line